### PR TITLE
Mate 1.26 bumps part 4

### PR DESCRIPTION
**Pull request by oz123**

### CODE REVIEW

File: mate-base/mate-menus/mate-menus-1.26.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+IUSE="debug +introspection nls"
+
+COMMON_DEPEND=">=dev-libs/glib-2.50:2
+	introspection? ( >=dev-libs/gobject-introspection-0.6.7:= )
+"
+
+RDEPEND="${COMMON_DEPEND}"
+
+DEPEND="${COMMON_DEPEND}
+	>=sys-devel/gettext-0.19.8
```
Comment (thesamesam) [inline mate-base/mate-menus/mate-menus-1.26.0.ebuild:27]: These are BDEPEND (gettext, pkgconfig).

File: mate-base/mate-panel/mate-panel-1.26.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	introspection? ( >=dev-libs/gobject-introspection-0.6.7:= )
+	wayland? ( gui-libs/gtk-layer-shell )
+"
+
+RDEPEND="${COMMON_DEPEND}
+	virtual/libintl
+"
+
+DEPEND="${COMMON_DEPEND}
+	app-text/docbook-xml-dtd:4.1.2
```
Comment (thesamesam) [inline mate-base/mate-panel/mate-panel-1.26.0.ebuild:52]: All of these (except for COMMON_DEPEND) are BDEPEND, except for maybe Perl.

File: mate-base/mate-session-manager/mate-session-manager-1.26.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	mate-base/mate-desktop
+	virtual/libintl
+	x11-apps/xdpyinfo
+	x11-misc/xdg-user-dirs
+	x11-misc/xdg-user-dirs-gtk
+	gnome-keyring? ( gnome-base/gnome-keyring )
+"
+
+DEPEND="${COMMON_DEPEND}
+	>=dev-lang/perl-5
```
Comment (thesamesam) [inline mate-base/mate-session-manager/mate-session-manager-1.26.0.ebuild:56]: All of these, except for maybe Perl (and obviously COMMON_DEPEND), are BDEPEND.

File: mate-base/mate-settings-daemon/mate-settings-daemon-1.26.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+		>=media-sound/pulseaudio-0.9.15
+	)
+	smartcard? ( >=dev-libs/nss-3.11.2 )
+	sound? (
+		>=media-libs/libmatemixer-1.10
+		media-libs/libcanberra[gtk3]
+	)
+	>=sys-devel/gettext-0.19.8
+	virtual/libintl
+	virtual/pkgconfig
```
Comment (thesamesam) [inline mate-base/mate-settings-daemon/mate-settings-daemon-1.26.0.ebuild:53]: gettext and pkgconfig likely don't belogn in RDEPEND. They should be in BDEPEND.

File: mate-base/mate-menus/mate-menus-1.26.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+	virtual/pkgconfig
+"
+
+src_configure() {
+	# Do NOT compile with --disable-debug/--enable-debug=no as it disables API
```
Comment (thesamesam) [inline mate-base/mate-menus/mate-menus-1.26.0.ebuild:32]: Now I'm wondering if all MATE packages should not be doing --disable-debug?

Comment (thesamesam) [reply mate-base/mate-menus/mate-menus-1.26.0.ebuild:32]: ?

Comment (oz123) [reply mate-base/mate-menus/mate-menus-1.26.0.ebuild:32]: I meant: can this be a use flag?
But it's already a use flag, just not enabled by default.
As for your question, I don't know. Most of the ebuild where written by NP-Hardass and I am only humble servant.
I don't understand everything he did. I slowly get it, but the intervals between my updates are long ...

Comment (thesamesam) [reply mate-base/mate-menus/mate-menus-1.26.0.ebuild:32]: What I'm saying is: this package has some special logic for debug. We should, at some point, look into whether other MATE packages should copy this logic.

No need to worry about it right now if you're not sure. :)

**Pull request closed without merging**